src/main/java/com/google/devtools/build/lib/analysis/PlatformConfiguration.java

```diff
@@ -60,6 +62,17 @@ public Label getHostPlatform() {
     return extraExecutionPlatforms;
   }
 
+  /**
+   * Returns the single target platform used in this configuration. The flag is multi-valued for
+   * future handling of multiple target platforms but any given configuration should only be
+   * concerned with a single target platform.
+   */
+  @Nullable
+  @Override
+  public Label getTargetPlatform() {
+    return Iterables.getFirst(targetPlatforms, null);
```

Comment:
Is this intended to return the first of possibly multiple, or does it really imply there's just a single target platform? The javadoc implies to me the latter, in which case would Iterables.getOnlyElement be a better fit?

Reply:
Good point: I've actually moved the logic to force a single platform much earlier in the configuration loading logic.